webmention.php

```diff
@@ -81,13 +114,34 @@ public static function parse_query($wp) {
     if (!array_key_exists('webmention', $wp->query_vars)) {
       return;
     }
-
-    $content = file_get_contents('php://input');
-    parse_str($content);
+    else {
+      // check if the end point has expired
+      $valid_ticks = array( 0, -1, -2 );
```

Comment:
I like the thinking, but the `` from https://core.trac.wordpress.org/browser/tags/4.1/src/wp-includes/pluggable.php#L1755 makes it impossible as far as I can see to make the built in `wp_create_nonce()` stateless as far as I can see. `'nonce_user_logged_out'` doesn't solve the stateless part – it was intended to increase statefulness rather than decreasing it so as long as there's a session with a token it will be impossible to make `wp_create_nonce()` stateless even with it.